server/swap/swap.go

```diff
@@ -1818,26 +1642,17 @@ func (s *Swapper) processInit(msg *msgjson.Message, params *msgjson.Init, stepIn
 	// Send the 'audit' request to the counter-party.
 	log.Debugf("processInit: sending contract 'audit' request to counterparty %v (%s) "+
 		"for match %v", ack.user, makerTaker(ack.isMaker), matchID)
-
-	// Register that there is an outstanding ack request. This is unregistered
-	// on completion of processAck or request expiry.
-	s.setLiveAcker(notification, ack)
-
-	// Expire function to unregister the outstanding request.
-	expireFunc := func() {
-		s.rmLiveAckers(notification.ID)
-		log.Infof("Timeout waiting for contract 'audit' request from user %v (%s) for match %v",
-			ack.user, makerTaker(ack.isMaker), matchID)
-	}
-
-	// Send the ack request.
+	// Send the request.
 
 	// The counterparty will audit the contract by retrieving it, which may
 	// involve them waiting for up to the broadcast timeout before responding,
 	// so the user gets at least s.bTimeout to the request.
-	s.authMgr.RequestWhenConnected(ack.user, notification, func(_ comms.Link, resp *msgjson.Message) {
+	s.authMgr.RequestWithTimeout(ack.user, notification, func(_ comms.Link, resp *msgjson.Message) {
 		s.processAck(resp, ack) // resp.ID == notification.ID
-	}, s.bTimeout, auth.DefaultConnectTimeout, expireFunc)
+	}, s.bTimeout, func() {
+		log.Infof("Timeout waiting for contract 'audit' request acknowledgement from user %v (%s) for match %v",
+			ack.user, makerTaker(ack.isMaker), matchID)
```

Comment:
Looks like we'll be getting a lot of these since clients are no longer required to send acks. Do you plan to subsequently retire this wait for ack feature for match-related requests? Basically, just send the request, if it doesn't get to the user, the match state can be recovered on connect.

Reply:
I was thinking this was something we'd want in the logs. I do suspect that we'll see a few of these, but hopefully not too many. Are you thinking `Debugf`?